tasks/src/text-generation/about.md

```diff
@@ -70,13 +70,34 @@ await inference.conversational({
 })
 ```
 
+## Text Generation Inference
 
+[Text Generation Inference (TGI)](https://github.com/huggingface/text-generation-inference) is a toolkit developed by Hugging Face to address challenges on response time and latency for concurrent users in serving large language models. It's an open-source serving solution that powers inference solutions of Hugging Face, [Inference Endpoints](https://huggingface.co/inference-endpoints), and [Inference API](https://huggingface.co/docs/api-inference/index) and [Hugging Chat](https://huggingface.co/chat/). You can use it to deploy any open-source large language model of your choice.
+
+## ChatUI Docker Spaces
+
+Hugging Face Spaces includes Docker templates to easily deploy your own instance of a specific application. [ChatUI](https://github.com/huggingface/chat-ui), is an open-source interface that enables serving conversational interface for large language models, and is currently available for Docker Spaces. The Docker template for ChatUI is powered by TGI for faster inference. Thanks to the template, you can deploy own instance based on a large language model with only a few clicks and customize it. Learn more about it [here](https://huggingface.co/docs/hub/spaces-sdks-docker-chatui) and create your large language model instance [here](https://huggingface.co/new-space?template=huggingchat/chat-ui-template).
```

Comment:
This might have too much advanced jargon. E.g. I don't expect most of our audience to know what Docker or Spaces is. Maybe the language here can be made a bit more beginner-friendly

Reply:
I iterated on it yesterday, can you check again if possible?